test/PublishCommand.js

```diff
@@ -204,6 +204,85 @@ describe("PublishCommand", () => {
         }
       }));
     });
+
+    it("should use semver increments when passed to repoVersion flag", (done) => {
```

Comment:
Sounds interesting, but not sure I understand clearly.  Can you give an example of how you'd like to see it organized?